datetime/example_test.go

```diff
@@ -79,3 +79,24 @@ func Example() {
 	fmt.Printf("Code: %d\n", resp.Code)
 	fmt.Printf("Data: %v\n", respDt.ToTime())
 }
+
+// Example demonstrates how to create a datetime for Tarantool without UTC
+// timezone in datetime.
+func ExampleNewDatetime_noTimezone() {
+	var datetime = "2013-10-28T17:51:56.000000009Z"
+	tm, err := time.Parse(time.RFC3339, datetime)
+	if err != nil {
+		fmt.Printf("error in time.Parse() is %v", err)
+		return
+	}
+
+	tm = tm.In(NoTimezone)
```

Comment:
So now I couldn't create a datetime without specifying timezone: I should either set a valid timezone or set NoTimezone timezone?

Reply:
GoLang by default sets UTC as timezone, unless you specify something else. So, yes. We need a value to specify a time without any timezone.

NoTimezone == "" (I've made a change) is a value for that purpose.